modules/tensor_mechanics/src/postprocessors/NodesetForce.C

```diff
@@ -0,0 +1,66 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NodesetForce.h"
+#include "MooseMesh.h"
+#include "SubProblem.h"
+
+registerMooseObject("MooseApp", NodesetForce);
+
+defineLegacyParams(NodesetForce);
+
+InputParameters
+NodesetForce::validParams()
+{
+  InputParameters params = NodalVariablePostprocessor::validParams();
+  params.set<bool>("unique_node_execute") = true;
+  params.addParam<std::string>("vector_tag", "TagName", "Tag Name this Aux works on");
+  params.set<ExecFlagEnum>("execute_on", true) = {EXEC_TIMESTEP_END};
+  params.addClassDescription(
+      "Computes the force in direction j=x|y|z on a nodeset from the disp_j");
```

Comment:
Suggested re-wording:
Computes the sum of the force in direction j=x|y|z on a nodeset from the residual corresponding with disp_j